src/include/duckdb.h

```diff
@@ -47,10 +47,22 @@ typedef enum DUCKDB_TYPE {
 
 typedef struct {
 	int16_t year;
-	int8_t month;
-	int8_t day;
+	int16_t month;
```

Comment:
Is there any reason why month, day, hour, min or sec should be int16_t instead of int8_t?

Reply:
Not really, I've corrected. There was a problem in the conversion bit it is solved now.